vllm/config.py

```diff
@@ -309,13 +309,15 @@ def __init__(
         cache_dtype: str,
         sliding_window: Optional[int] = None,
         enable_prefix_caching: bool = False,
+        is_openvino_cpu: bool = False
```

Comment:
I suppose it's not allowed to change public vLLM classes.
can we handle it internally in OpenVINO entities?

Reply:
Removed changes in public vLLM classes, use environment var `VLLM_OPENVINO_CPU_KV_CACHE_PRECISION=u8` to enable it.